.buildkite/plugins/factory-reporter/hooks/post-command

```diff
@@ -12,27 +12,36 @@ if [[ $SKIP_BUILDKITE_PLUGINS == "true" ]]; then
     echo "SKIP_BUILDKITE_PLUGINS is set. Skipping factory reporter"
     exit 0
 fi
+if [[ $BUILDKITE_PULL_REQUEST != "false" ]]; then
+    exit 0
+fi
 if [[ -z $PIPELINE ]]; then
     echo "No pipeline ID found, skipping factory reporter"
     exit 0
 fi
 
-if [[ $BUILDKITE_PULL_REQUEST == "false" ]]; then
-
-  START_SECONDS=$(buildkite-agent meta-data get start-seconds)
-  if [[ -z $START_SECONDS ]]; then
-    echo "Error: start-seconds not set as meta-data" >&2
-    exit 1
-  fi
-
-  FACTORY_COMMAND=$(buildkite-agent meta-data get factory-command)
-  if [[ -z $FACTORY_COMMAND ]]; then
-    echo "Error: factory-command not set as meta-data" >&2
-    exit 1
-  fi
-  echo "Using factory command: $FACTORY_COMMAND"
+START_SECONDS=$(buildkite-agent meta-data get start-seconds)
+if [[ -z $START_SECONDS ]]; then
+  echo "Error: start-seconds not set as meta-data" >&2
+  exit 1
+fi
 
+FACTORY_COMMAND=$(buildkite-agent meta-data get factory-command)
+if [[ -z $FACTORY_COMMAND ]]; then
+  echo "Error: factory-command not set as meta-data" >&2
+  exit 1
```

Comment:
I'm conflicted if we should exit with 1 in there two cases, afterall yes it's a "bad condition", but... 🤷🏼

Reply:
This means something that should happen did not happen in a previous step, so I think it's a failure